config/l3afd.cfg

```diff
@@ -42,8 +44,8 @@ is-user-program: false
 [tc-root-program]
 name: tc_root
 artifact: l3af_tc_root.tar.gz
-ingress-map-name: /sys/fs/bpf/tc/globals/tc_ingress_root_array
-egress-map-name: /sys/fs/bpf/tc/globals/tc_egress_root_array
+ingress-map-name: tc_ingress_root_array
+egress-map-name: tc_egress_root_array
```

Comment:
did you want to leave "tc/globals" on the front of these?  Ok with me either way, but I see that "tc/globals" was added to lines 783, 845, etc. of bpf_test.go so I suspect you might want these two files to be consistent.

Reply:
yes